addons/stock/security/ir.model.access.csv

```diff
@@ -27,8 +27,7 @@ access_product_group_res_partner_stock_manager,res_partner group_stock_manager,b
 access_product_pricelist_item_stock_manager,product.pricelist.item stock_manager,product.model_product_pricelist_item,stock.group_stock_manager,1,1,1,1
 access_stock_warehouse_orderpoint,stock.warehouse.orderpoint,model_stock_warehouse_orderpoint,stock.group_stock_user,1,0,0,0
 access_stock_warehouse_orderpoint_system,stock.warehouse.orderpoint system,model_stock_warehouse_orderpoint,stock.group_stock_manager,1,1,1,1
-access_stock_quant_manager,stock.quant manager,model_stock_quant,stock.group_stock_manager,1,0,0,0
-access_stock_quant_user,stock.quant user,model_stock_quant,stock.group_stock_user,1,0,0,0
+access_stock_quant_manager,stock.quant manager,model_stock_quant,stock.group_stock_manager,1,1,1,0
```

Comment:
I would keep stock user since they can do inventory adjustement now

Reply:
I removed it because there is the same access for internal users two lines below